src/actors/upgrade/network_status/actors/netifacescanner/actor.py

```diff
@@ -0,0 +1,125 @@
+from leapp.actors import Actor
+from leapp.models import IfaceResult, IfacesInfo
+from leapp.tags import ScanTag
+import subprocess
+import os
+import re
+import sys
+
+
+class NetIfaceScanner(Actor):
+    name = 'net_iface_scanner'
+    description = 'No description has been provided for the net_iface_scanner actor.'
+    consumes = ()
+    produces = (IfaceResult,)
+    tags = (ScanTag,)
+
+    def process(self):
+        iface_info = self.return_ifs_info()
+        self.produce(iface_info)
+        self.log.info("Finished scanning network interfaces")
+        pass
+
+    def get_ifaces_names(self, sys_command):
```

Comment:
The `sys_command` parameter doesn't make sense to me when there is currently just one command that is processed. Is there known any next possible use of this function? I suggest to put the `sys_command` into the function and remove the parameter completely. In addition, you expect that output of the command is "list" of directories. In that case more relevant as an input would be a list of directories with which the function should work. But now the best solution would be move of the sys_command into the function directly.